### PR TITLE
completed javascript functions

### PR DESCRIPTION
**Pull request by salgarcia666**
sal garcia: completed javas
Screenshot (116)
cript functions

### CODE REVIEW

File: javascript-functions/main.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+var object = {
+  firstName: 'Cash',
+  lastName: 'Money'
+};
+
+console.log(getFirstName(object));
+
+function getLastElement(array) {
+  var last = array.length;
```
Comment (Zhiwutian) [inline javascript-functions/main.js:34]: this is close, but you don't want the length of the array, you want to return the last item in the array.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update!

File: javascript-functions/main.js (hunk 1 above)
Comment (salgarcia666) [reply javascript-functions/main.js]: Screenshot (117)

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**